CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ All notable changes to this project will be documented in this file.
   (<https://github.com/openvinotoolkit/training_extensions/pull/3759>)
 - Enable to use polygon and bitmap mask as prompt inputs for zero-shot learning
   (<https://github.com/openvinotoolkit/training_extensions/pull/3769>)
+- Add num_devices in Engine for multi-gpu training
+  (<https://github.com/openvinotoolkit/training_extensions/pull/3778>)
```

Comment:
I think it'd be better to be moved into `Bug fixes`.
The ticket was set as a bug and this PR seems to fix that multi-gpu settings are unavailable through Engine.
What do you think?

Reply:
Thanks! i updated CHANGELOG - https://github.com/openvinotoolkit/training_extensions/pull/3778/commits/b6ad495c275325a80aebc80278fd518103e30518